files/chefdk-env-customization/windows/chefdk_env_customization.rb

```diff
@@ -0,0 +1,52 @@
+## Environment hacks for running Ruby with ChefDK ##
+# ENV['HOME'] is not set by default on Windows. We need to set this to
+# something sensible since a lot of Ruby code depends on it. It is important 
+# for this directory to exist and be available, so we are introducing logic
+# here to pick a working HOME
+#
+# You can find this file in the repo at https://github.com/chef/omnibus-chef
+
+if !ENV['HOME'] || !File.exists?(ENV['HOME'])
+  old_home = ENV['HOME']
+  found = false
+  alternate_homes = []
+  alternate_homes << "#{ENV['HOMEDRIVE']}#{ENV['HOMEPATH']}" if ENV['HOMEDRIVE']
+  alternate_homes << "#{ENV['HOMESHARE']}" if ENV['HOMESHARE']
```

Comment:
http://windowsitpro.com/windows/jsi-tip-2965-homeshare-and-homepath-differences-between-windows-2000-and-windows-nt-40

This indicates that before Windows 2000, it was %HOMESHARE%%HOMEPATH% but in windows 2000 it became just %HOMESHARE%. :question: